lib/railway_ipc/ipc/consumers/republished_messages_consumer.ex

```diff
@@ -0,0 +1,30 @@
+defmodule RailwayIpc.Ipc.RepublishedMessagesConsumer do
+  use RailwayIpc.CommandsConsumer,
```

Comment:
maybe create a republish commands consumer that knows its publish function and doesnt utilize commands_exchange vs. events_exchange